torch/_dynamo/testing.py

```diff
@@ -246,24 +245,33 @@ def __call__(
         return gm.forward
 
 
-# Equivalent to backend="aot_eager", but also records graphs that
-# we can assert on
-class AOTEagerAndRecordGraphs:
+class AotEagerAndRecordGraphs:
     def __init__(self) -> None:
         self.graphs: List[torch.fx.GraphModule] = []
+        self.fw_graphs: List[torch.fx.GraphModule] = []
+        self.bw_graphs: List[torch.fx.GraphModule] = []
```

Comment:
Er, I think what happened is that https://github.com/pytorch/pytorch/pull/137677 added one and then this PR is merge conflicting

Reply:
Yes.